platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/MigrationCommand.java

```diff
@@ -41,7 +42,7 @@
 
   @Reference protected ConfigurationMigrationService configurationMigrationService;
 
-  protected Path defaultExportDirectory =
+  protected Path exportDirectory =
```

Comment:
:question: Does this mean that we removed the ability for an admin to specify an alternate export directory?

Reply:
correct. Because of the security manager, the admin would have to first modify the policy file to allow us read and write access to the new directory. Since we are now simply generating a zip file, we keep it simpler.